flight/Modules/Sensors/simulated/simsensors.c

```diff
@@ -85,7 +86,9 @@ static float rand_gauss();
 
 static bool use_real_sensors;
 
-enum sensor_sim_type {CONSTANT, MODEL_AGNOSTIC, MODEL_QUADCOPTER, MODEL_AIRPLANE, MODEL_CAR} sensor_sim_type;
+extern bool use_yasim;
```

Comment:
Yah.  It's like posix needs some kind of configuration registry for things like this--- ultimately moving away from the command line to a config file and adding a small API would help for this.